src/Microsoft.ML.TensorFlow/TensorFlowModel.cs

```diff
@@ -40,7 +43,7 @@ internal TensorFlowModel(IHostEnvironment env, Session session, string modelLoca
         /// </summary>
         public DataViewSchema GetModelSchema()
         {
-            return TensorFlowUtils.GetModelSchema(_env, Session.graph);
+            return TensorFlowUtils.GetModelSchema(_env, Session.graph, treatOutputAsBatched: TreatOutputAsBatched);
```

Comment:
Nit: no need to use named parameter here, since the names are the same.

Reply:
Fixed by making that parameter be a normal parameter, so the name is no longer needed.